<?xml version="1.0"?>
<launch>
  <node pkg="greenship_pathfinder" 
        type="autopath_core_starter.py" 
        name="autopath_core" 
        output="screen">
    <param name="log_level" value="debug"/>
    <env name="ROS_LOG_LEVEL" value="debug"/>
    <!-- Enable debug output for all ROS logs -->
    <env name="ROSCONSOLE_FORMAT" value="[${severity}] [${time}] ${node}: ${message}"/>
  </node>
</launch>